config/opendatahub-io/org.yaml

```diff
@@ -520,6 +520,7 @@ orgs:
       adr-maintainers:
         description: Maintainers of the ADR repository
         maintainers:
+	- etirelli
```

Comment:
Since the rest of the file is using spaces for indentation, can you change this to spaces for uniformity

Reply:
@LaVLaS done